### ISSUE TITLE
Grouped product status nog syncing

### ISSUE THREAD

**Issue by MatthijsvanNoort** [akeneo/magento2-connector-community#677]
The product status of grouped products is not synced

### Environment and configuration
Make grouped family code and set mapping in connector
Grouped product family code: 'GroupedProductsOnly'
Quantity association code: 'Grouped_Product'

### Steps to reproduce
1. Create a grouped product in Akeneo, disabled
2. Edit some info, add some associations, enable product, save
3. Run sync jobs

### Expected result
Grouped Product data is synced
Grouped Product is enabled in Magento

### Actual result
Grouped Product data is synced
Product remains disabled in Magento

**Comment by dennisderoo** [comment]
This is still not working in version 104.3.17

And I think, I found the reason.

When you look in /Job/Product.php
The product is saved as type grouped (which is correct)
Line 1028:
`$connection->addColumn(
                $tmpTable,
                '_type_id',
                [
                    'type' => 'text',
                    'length' => 255,
                    'default' => 'grouped',
                    'COMMENT' => ' ',
                    'nullable' => false,
                ]
            );`

However, "grouped" is not a allowed type_id (list of allowed type_id that can be imported) :
Line 146:
`protected $allowedTypeId = ['simple', 'virtual'];`

Because of this, it will not update the correct status, as 'grouped' is not within the allowedTypeId
Line 2279:
`$connection->update(
            $tmpTable,
            ['_status' => $status],
            ['_is_new = ?' => 1, '_status = ?' => 1, '_type_id IN (?)' => $this->allowedTypeId]
        );`

**Comment by cschwerdt** [comment]
We’ve had the same experience with importing grouped products. Have you already tried a workaround? Is it enough to just add “grouped” to the list?

**Comment by dennisderoo** [comment]
> We’ve had the same experience with importing grouped products. Have you already tried a workaround? Is it enough to just add “grouped” to the list?

Yes, adding "grouped" in the list from $allowedTypeId worked for us.
